test/errorhandler.js

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var app;
+var assert = require('assert');
+var request = require('supertest');
+var util = require('util');
+var async = require('async');
+var app = require('../');
+var loopback = require('loopback');
+var errorHandler = require('strong-error-handler');
+var boot = require('loopback-boot');
+
+describe('loopback.errorHandler(options)', function() {
+  it('should delete stack when options.debug is false', function(done) {
```

Comment:
Honestly, I don't see much value in testing how loopback-workspace treat HTTP error responses.

We need a unit-test verifying behaviour of scaffolded apps, as I described here: https://github.com/strongloop/loopback-workspace/pull/284#discussion_r66055596